cvss/src/cvss3.rs

```diff
@@ -24,6 +25,24 @@ impl Display for Cvss3Base {
     }
 }
 
+impl Serialize for Cvss3Base {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serializer.collect_str(self)
+    }
+}
+
+impl<'de> Deserialize<'de> for Cvss3Base {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let s = String::deserialize(deserializer)?;
+        FromStr::from_str(&s).map_err(de::Error::custom)
+    }
+}
```

Comment:
Can this not be derived? Just a string, right?

Reply:
I dunno. It's what StackOverflow told me when I asked how to use `Display` as a serialization. The actual object is a multi-field struct.

Reply:
ah. normally i distrust code comments, but in this case, i'm not sure i could deduce that intent from this block. and now i'm curious: why do you need to use `Display` as a serialization?

Reply:
Because the CVSS score is actually communicated standardly as a string, but in our DB and entities, we break out its components into columns for search-ability.

Reply:
personally, i'd prefer `.to_string()` instead.